source/operating-a-service/incident-playbook.html.md

```diff
@@ -64,43 +73,47 @@ Contact your users if:
 
 Informing users about incidents is generally considered best practice, but should be decided on a case by case basis with the product and service managers.
 
-### 8. If the incident requires invocation of Disaster Recovery procedures
+The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). For some incidents, you might find it helpful to update this. You will need a GitHub account to do this, or delegate updates to a colleague who has one.
 
-Follow the [Disaster Recovery procedure](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/disaster-recovery.md)
+The updates are managed via GitHub Actions and issues on the [teacher-services-upptime repository on GitHub](https://github.com/DFE-Digital/teacher-services-upptime). If a service’s automatic health check is failing continuously, an issue will be created within 5 minutes of the failure occurring and the dashboard will start reporting a service issue.
 
-## While the incident is in progress (all incident leads)
+To update the dashboard:
 
-Keep all conversations and status updates about the incident on the dedicated Slack incident channel.
+1. Navigate to the appropriate incident issue on the [GitHub issues page](https://github.com/DFE-Digital/teacher-services-upptime/issues)
+2. Add a comment to the issue
+
+### 9. If the incident requires invocation of Disaster Recovery procedures
+
+Follow the [Disaster Recovery procedure](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/disaster-recovery.md)
 
-Use these incident stages in your Slack updates:
+## Provide updates during the incident (usually comms lead)
 
-- Incident has occurred
-- Incident is being assessed
-- Incident is being fixed
-- Incident is resolved
+Keep all conversations and status updates about the incident on the dedicated Slack incident channel.
 
 Use the Slack IncidentBot `/incident update` command to update:
 
 - Description
 - Priority
 - Leads
 
-### Provide regular updates every 60 minutes (comms lead)
+Update stakeholders on the Slack incident channel everytime something changes or everyday, until the incident has been resolved. 
```

Comment:
```suggestion
Update stakeholders on the Slack incident channel regularly, until the incident has been resolved. This might be every hour for high risk incidents, but at least every time something changes. Make sure your team is kept updated too by linking them to the incident channel.
```

Reply:
looks good